### ISSUE TITLE
Rotated MNIST example fails if the device is 'cuda'

### ISSUE THREAD

**Issue by lupoglaz** [QUVA-Lab/e2cnn#3]
Fix:
cell 7: 
instead of
x_transformed.to(device) 
this:
x_transformed = x_transformed.to(device)

cell 9:
instead of: 
loss = loss_function(y, t)
this:
loss = loss_function(y, t.to(device=device))

instead of:
correct += (prediction == t).sum().item()
this:
correct += (prediction == t.to(device=device)).sum().item()

**Comment by Gabri95** [comment]
Hi @lupoglaz ,

I only tested that script on my laptop without GPU.

I'll fix this, thank you very much for reporting this problem!
